CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 ### Payments
 * [Fixed] STPPaymentHandler.handleNextAction allows payment methods that are delayed or require further customer action like like SEPA Debit or OXXO.
 
+### PaymentSheet
+* [Added] Improved validation for ephemeral keys
```

Comment:
Does this need a changelog entry? This is not a user facing change.

In case we do need the entry, tiny nit:
```suggestion
* [Added] Improved validation for ephemeral keys.
```

Reply:
I'm not sure what we normally do/do not include in the changelog. It does improve user-facing error messages that users have complained about in the past, so sorta-kinda user facing?

Reply:
I can go either way with this one. If users have specifically mentioned this then it is worth calling it out.